### PR TITLE
test: fixed test cases of potential duplicates filter

### PR DESCRIPTION
**Pull request by devendrafyle**
## Clickup
https://app.clickup.com/t/86cw7ngz0

### CODE REVIEW

**Comment by suyashpatil78** [comment]
@devendrafyle unit tests are still breaking. Are you fixing those in incremental PRs?

**Comment by suyashpatil78** [comment]
Ohh okay, can ignore. that is the percentage which is failing

File: src/app/core/mock-data/expense-filters.data.ts [hunk 1]
```diff
@@ -16,6 +16,30 @@ export const expenseFiltersData1: Partial<ExpenseFilters> = deepFreeze({
...
+  receiptsAttached: 'YES',
+  type: ['PerDiem', 'Mileage'],
+  sortParam: 'category->name',
+  sortDir: 'asc',
+  cardNumbers: ['1234', '5678'],
+  splitExpense: 'YES',
+  potentialDuplicates: 'YES',
+});
+
+export const expenseWithoutPotentialDuplicateFilterData: Partial<ExpenseFilters> = deepFreeze({
```
Comment (suyashpatil78) [inline src/app/core/mock-data/expense-filters.data.ts:31]: can we reuse `expenseWithPotentialDuplicateFilterData` here and extend it?

File: src/app/core/services/platform/v1/shared/expenses.service.spec.ts [hunk 2]
```diff
@@ -568,6 +570,24 @@ describe('ExpensesService', () => {
...
+  describe('generatePotentialDuplicatesParams():', () => {
+    it('should generate params for potential duplicates expenses ', () => {
+      const result = service.generatePotentialDuplicatesParams({}, expenseWithPotentialDuplicateFilterData);
+
+      expect(result).toEqual({
+        is_duplicate_present: 'eq.true',
+      });
+    });
+
+    it('should generate params no potential duplicate expenses', () => {
```
Comment (suyashpatil78) [inline src/app/core/services/platform/v1/shared/expenses.service.spec.ts:582]: `should generate params for non-potential duplicate expenses`

**Review by suyashpatil78 — APPROVED** [review]
2 comments, rest LGTM

**Review by Chethan-Fyle — APPROVED** [review]

File: src/app/core/mock-data/expense-filters.data.ts (hunk 1 above)
Comment (devendrafyle) [reply src/app/core/mock-data/expense-filters.data.ts:31]: haa we can, updating it

File: src/app/core/services/platform/v1/shared/expenses.service.spec.ts (hunk 2 above)
Comment (devendrafyle) [reply src/app/core/services/platform/v1/shared/expenses.service.spec.ts:582]: sure will update this

**Pull request merged**